main_test.go

```diff
@@ -54,14 +54,13 @@ func TestTreasurySetup(t *testing.T) {
 	})
 
 	t.Run("Treasury should be able to receive non-fungible tokens", func(t *testing.T) {
-		otu.CreateNFTCollection("account")
-		otu.MintNFT("account")
+		otu.CreateNFTCollection("signer1")
+		otu.MintNFT("signer1")
```

Comment:
whats the purpose of changing up the account names here?

Reply:
I think I got lost in tests, will revert to account and then see if it still works :D once the previous PR is merged I'll update the changes

Reply:
sounds good! this might all work, just seemed unnecessarily confusing

Reply:
Only signers can deposit NFT collections, so I had to update it after all to signer1